### PR TITLE
Add Frintx_S, ASRV test, update ADCS, use Assert.Multiple and indent

### PR DESCRIPTION
**Pull request by MS-DOS1999**

### CODE REVIEW

**Comment by gdkchan** [comment]
fyi @MerryMage

File: Ryujinx.Tests/Cpu/CpuTestAlu.cs [hunk 1]
```diff
@@ -17,20 +17,24 @@ public void Adc(uint Opcode, uint A, uint B, bool CarryState, uint Result)
...
-        [TestCase(0xBA020020u, 2u,          3u,   false, false, false, 5u)]
-        [TestCase(0xBA020020u, 2u,          3u,   true,  false, false, 6u)]
-        [TestCase(0x3A020020u, 0xFFFFFFFEu, 0x1u, true,  true,  true,  0x0u)]
-        public void Adcs(uint Opcode, uint A, uint B, bool CarryState, bool Zero, bool Carry, uint Result)
+        [TestCase(0x3A020020u, 2u,          3u,          false, false, false, false, 5u)]
+        [TestCase(0x3A020020u, 2u,          3u,          true,  false, false, false, 6u)]
+        [TestCase(0xBA020020u, 2u,          3u,          false, false, false, false, 5u)]
+        [TestCase(0xBA020020u, 2u,          3u,          true,  false, false, false, 6u)]
+        [TestCase(0x3A020020u, 0xFFFFFFFEu, 0x1u,        true,  false, true,  true,  0x0u)]
+        [TestCase(0x3A020020u, 0xFFFFFFFFu, 0xFFFFFFFFu, true,  true, false, true,  0xFFFFFFFFu)]
```
Comment (MerryMage) [inline Ryujinx.Tests/Cpu/CpuTestAlu.cs]: nit: spacing is a bit weird here

File: Ryujinx.Tests/Cpu/CpuTestSimdArithmetic.cs [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    	[TestCase(0x7FC00002u, 'N', false, 0x7FC00002u)]
+    	[TestCase(0x7FC00002u, 'P', false, 0x7FC00002u)]
+    	[TestCase(0x7FC00002u, 'M', false, 0x7FC00002u)]
+    	[TestCase(0x7FC00002u, 'Z', false, 0x7FC00002u)]
+    	[TestCase(0x7FC00002u, 'N', true,  0x7FC00000u)]
+    	[TestCase(0x7FC00002u, 'P', true,  0x7FC00000u)]
+    	[TestCase(0x7FC00002u, 'M', true,  0x7FC00000u)]
+    	[TestCase(0x7FC00002u, 'Z', true,  0x7FC00000u)]
+    	public void Frintx_S(uint A, char RoundType, bool DefaultNaN, uint Result)
+        {
```
Comment (MerryMage) [inline Ryujinx.Tests/Cpu/CpuTestSimdArithmetic.cs]: nit: Indentation in this function is kinda funky.

**Comment by gdkchan** [comment]
lgtm, thanks!

**Pull request merged**